libs/content/workspace/src/storage_viewer/show.rs

```diff
@@ -0,0 +1,361 @@
+use super::data::{self, NodeLayer};
+use color_art;
+use colors_transform::{self, Color};
+use egui::{
+    self, menu, Align2, Color32, FontFamily, FontId, Id, LayerId, Pos2, Rect, Rounding, Sense,
+    Stroke, TextWrapMode, Ui,
+};
+use lb_rs::blocking::Lb;
+use lb_rs::model::file::File;
+use lb_rs::model::usage::bytes_to_human;
+use lb_rs::Uuid;
+
```

Comment:
This style of comments is generally done with a `/// comment`, it'll show up in tooling as people use it.